compiler/docgen.nim

```diff
@@ -508,19 +498,48 @@ proc prepareExample(d: PDoc; n: PNode): tuple[rdoccmd: string, code: string] =
     if n1.kind notin nkStrKinds: globalError(d.conf, n1.info, "string litteral expected")
     rdoccmd = n1.strVal
 
-  var docComment = newTree(nkCommentStmt)
+  let useRenderModule = false
```

Comment:
So ... there is a bunch of dead code here regarding `useRenderModule`. Why?

Reply:
this would snowball into making `renderModule` itself deadcode, which may (or not) be a problem
How about i defer to another PR removing this deadcode, and decide there whether to also remove `renderModule` or add some test for it?

Reply:
Sure, ok.